scripts/tools-versions-linux.txt

```diff
@@ -0,0 +1,9 @@
+python=not settled
+git=not settled
+cmake=not settled
+ninja=not settled
+gperf=not settled
+dtc=not settled
```

Comment:
I propose:
```suggestion
python=3.8.0
git=2.17.1
ninja=1.8.2
gperf=3.1
```
as those can be installed on Ubuntu 18.04 using `apt-get install` directly.

DTC on my system is from Zephyr SDK, and CMake is manually installed, to keep up-to-date on latest versions.

Reply:
The versions can be pinned for apt-get (snapcraft stage uses this too) only by full version spec:
```
git=1:2.17.1-1ubuntu0.7
ninja-build=1.8.2-1
gperf=3.1-1
ccache=3.4.1-1
dfu-util=0.9-1
python3.8=3.8.0-3~18.04
libpython3.8-dev=3.8.0-3~18.04
```
Cmake from the kitware repo is available in the range of 3.15.0 to 3.17.x, since there's no 3.13 used by win/mac, probably the latest is alright.

Reply:
Please add `cmake=3.17.3-0kitware1` and `dtc=1.4.7-3ubuntu2`.
Full working config: https://gist.github.com/bencefr/6f492bf8fdd851faacafbfcd08ba538a#file-tools-versions-linux-txt